src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,22 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => (
+  <ul>
```

Comment:
add the same logic for selecting `activeTab` as i suggested doing in `App.jsx` to prevent tests from failing
```suggestion
export const Tabs = ({ tabs, activeTabId, onTabSelected }) =>{
  const activeTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
  return  (
  <ul>
```